blogposts/2021/creating-sourcegraph-product-design-principles.md

```diff
@@ -0,0 +1,132 @@
+
+As we’ve grown over the last few months, the Sourcegraph design and product teams have found ourselves having more discussions around product and design decisions. 
+This is a common challenge for design teams everywhere: there are often many valid ways to solve a problem, each with its own set of tradeoffs and assumptions. 
+A small team can rely on intuition, experience, and shared trust to make decisions in these moments. **But gut instinct doesn’t scale.**
+
+The design and product teams were at an inflection point. That’s why we decided it was time to come together to define our product design principles. 
+I’m going to go into how we did this and what principles we settled on, but first I’ll cover some basics:
+
+
+## What are design principles?
+
+Design principles are a set of statements co-created by the team to express a **shared vision and values** for the product.
+
+A strong set of design principles gives the team a way to make **consistent decisions** based on a shared idea of what we’re trying to achieve. 
+They **introduce constraints** that help reduce ambiguity, particularly when there’s more than one possible solution to a given problem. They also give us a way to **capture our commitment to inclusion** at every step of the design process.
+
+
+## Who uses design principles?
+
+While they might be called design principles, they’re for everyone in the organization.
+
+Designers use design principles to guide their explorations and outcomes. Product managers use design principles to shape priorities and features. 
+Developers use design principles as a lens to collaborate on design, from discovery to delivery.
+
+
+### Sidebar: What’s the difference between design principles and design patterns?
+
+Design principles and design patterns are often conflated. Where design principles are simple, broadly applicable, and capture our shared vision and values, design patterns are the tangible and repeatable expression of our principles in combination with heuristics of good design and usability. 
+
+Design principles don’t change often, as they reflect a core truth of who we are. Design patterns change over time to meet new needs, technology, and trends.
+
+
+## How we defined our design principles
+
+At Sourcegraph, we’re fully remote and globally distributed. Our product and design teams span 13 time zones. 
+That’s why we prioritize asynchronous collaboration together with thoughtful moments of real-time collaboration.
+
+We wanted to involve product designers, product managers, and engineers throughout the process to co-create our design principles. 
+To make sure we could create the best collaborative environment, we planned a process that would start asynchronously before bringing us together for a real-time workshop when we needed an immediate feedback loop.
+
+
+### The process
+
+
+#### 1. Aligning on our goals and conceptual understanding of principles
```

Comment:
@quinnkeast I'm not sure why these are rendering larger than "The process" – if you know how to correct it please do